drivers/hmis/app/graphql/schema.graphql

```diff
@@ -192,6 +192,7 @@ User account for a user of the system
 """
 type ApplicationUser {
   activityLogs(limit: Int, offset: Int): ActivityLogsPaginated!
```

Comment:
I'm curious about this "activity logs," is it related to the user edit audit functionality I'm adding? I can see later in the class that the activity logs are only returned if the user has `can_audit_users` permission, so maybe yes?
I think it would be good for me to understand better what the relationship is, then maybe I can add some comments clarifying

Reply:
Yeah good question! The tldr difference is one is "access" history (Activity Logs) and "edit" history (PaperTrail versions).  We use the generic "audit" term to encompass both types of history (view and edit). But many places in the code ,"audit" just refers to  edit history, so it can get confusing.

1) Access History
The `hmis_activity_logs` table in the app DB stores logs about every GraphQL request that was made. That is pretty raw data so it's not helpful to show directly to the user, so, we do some additional processing in the `Hmis::ActivityLogProcessorJob` that populates `hmis_activity_logs_clients` and `hmis_activity_logs_enrollments`. Those 3 tables are ultimately what back the "Access History" page for a given user, which lets you see which clients and enrollments that a given user has **viewed**.


2) Edit History
The `versions` table in the warehouse DB stores [PaperTrail](https://github.com/paper-trail-gem/paper_trail) change history for any record that has `has_paper_trail` specified on the model. Each time a Client record is changed (for example), a new "version" record is added that has the before-and-after changed fields, along with who made the change.

This table is what populates the "Client Audit History" and "Enrollment Audit History", as well as the view youre adding here.

Reply:
Great - thank you. I'll add some comments summarizing this to `user.rb`